COMMERCE-9744 - Subscription length always shows singular words

elocamp opened the pull request:
The language keys for time periods in SubscriptionTypeImpl files were not being passed in their plural form, and some didn't exist
Proposed fix is to modify the language keys for time periods to their adapted plural form and add the language keys that doesn't exist

Related Ticket: [COMMERCE-9744](https://issues.liferay.com/browse/COMMERCE-9744)

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

matthewchan1 commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest(master)/4157/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-7.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4157/jenkins-report.html">here</a>.</b></p>

matthewchan1 commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest(master)/4415/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-4.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4415/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 25 out of 27 jobs passed in 1 hour 27 minutes 30 seconds 685 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d6b23bb20fb1439a9415d3fd946dfa6159ce2315">d6b23bb20fb1439a9415d3fd946dfa6159ce2315</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/b186f390382e554f5ada1a667c464f572587fabb">b186f390382e554f5ada1a667c464f572587fabb</a></p>25 out of 27 jobs PASSED<div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest(master)/4157/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/311395/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/4</strong></strike></a></li></ul></div><details><summary><strong>25 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/256151/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/177802/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/303197/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/307736/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/177801/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/307735/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/277588/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/256154/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/277587/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/277586/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/256152/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/313190/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/177800/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/64651/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/313189/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/277585/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/7</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/289053/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/64650/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/64648/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/277589/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/311396/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/64649/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/64652/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/282459/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/256153/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-7.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4157/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/311395/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/4</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/311395//consoleText">AXIS_VARIABLE=0,label_exp=!master #311395</a><ol><li><div><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/311395//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PGStaging_ActivateRemoteStaging_">LocalFile.PGStaging#ActivateRemoteStaging</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-7/1563393348071/test-portal-acceptance-pullrequest(master)/4157/functional-tomcat90-mysql57-jdk8/4/0/LocalFile.PGStaging_ActivateRemoteStaging/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-7/1563393348071/test-portal-acceptance-pullrequest(master)/4157/functional-tomcat90-mysql57-jdk8/4/0/LocalFile.PGStaging_ActivateRemoteStaging/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-7/1563393348071/test-portal-acceptance-pullrequest(master)/4157/functional-tomcat90-mysql57-jdk8/4/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 25 out of 27 jobs passed in 1 hour 35 minutes 34 seconds 731 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d6b23bb20fb1439a9415d3fd946dfa6159ce2315">d6b23bb20fb1439a9415d3fd946dfa6159ce2315</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/b186f390382e554f5ada1a667c464f572587fabb">b186f390382e554f5ada1a667c464f572587fabb</a></p>25 out of 27 jobs PASSED<div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest(master)/4415/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/307739/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</strong></strike></a></li></ul></div><details><summary><strong>25 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/306574/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/304871/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/306580/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/327030/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/306579/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/327029/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323482/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/304870/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/327027/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/306577/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/327026/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/306578/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/327028/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323481/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/307737/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/306575/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/7</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/64653/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/306576/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/306572/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323483/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/306581/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/306573/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/307738/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/64654/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/323480/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-4.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4415/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/307739/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></h5><div><h6>Job Results:</h6><p>11 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/307739//consoleText">AXIS_VARIABLE=0,label_exp=!master #307739</a><ol><li><div><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/307739//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PGStaging_AssertNoEditLink_">LocalFile.PGStaging#AssertNoEditLink</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-4/1563394498925/test-portal-acceptance-pullrequest(master)/4415/functional-tomcat90-mysql57-jdk8/1/0/LocalFile.PGStaging_AssertNoEditLink/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-4/1563394498925/test-portal-acceptance-pullrequest(master)/4415/functional-tomcat90-mysql57-jdk8/1/0/LocalFile.PGStaging_AssertNoEditLink/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-4/1563394498925/test-portal-acceptance-pullrequest(master)/4415/functional-tomcat90-mysql57-jdk8/1/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></details></html>

brianikim commented:
Hello @matthewchan1 ,

Please investigate the test failure.

Sincerely,
Brian Kim

brianikim commented:
Closing this PR as we saw that the test failure is locally reproducible.

slnn committed:
LPS-168633 Remove "immediate = true" from module portal-osgi-web

slnn committed:
LPS-168633 Remove immediate=true from PortletTracker needs to set empty value for service otherwise, PortletTracker will never be activated

rafaprax committed:
LPS-169623 Add ability to check if the item should be added to collection

rafaprax committed:
LPS-169623 Create a helper class to hold a tracker of ModelResourcePermission and verify if a user has permission to view a given entity

rafaprax committed:
LPS-169623 Send notifications to users who have permission to view the entity

rafaprax committed:
LPS-169623 Baseline

Commit:
LPS-171231 KaleoTaskAssignmentSelectorRegistryImpl: Convert method reference to ServiceTrackerMap

Commit:
LPS-171231 Fix the test

carolmariaabb committed:
LPS-171052 Add new method to allow removing model resource actions

carolmariaabb committed:
LPS-171052 Implement new method

Commit:
LPS-171460 Convert method reference to ServiceTrackerList

Commit:
LPS-171460 Add dependencies

rafaprax committed:
LPS-169623 Use inheritance instead of composition approach

rafaprax committed:
LPS-169623 Create a registry class to ModelResourcePermission services

rafaprax committed:
LPS-169623 Apply to direct usages

rafaprax committed:
LPS-169623 Apply to ObjectEntryServiceImpl as well. Even though it uses a subset of the ModelResourcePermission ("(&(com.liferay.object=true)(model.class.name=*))"), it can leverage the registry logic since ObjectDefinitionModelResourcePermission is created only for non system objects and must not have more than one ModelResourcePermission service per class name.

julschong committed:
LPS-171358 Remove Stream usages in commerce-product-content-search-web and use ListUtil and TransformUtil instead

Commit:
LPS-171231 CompositeKaleoTaskAssignmentSelector: Replace method reference(addKaleoTaskAssignmentSelector) with class KaleoTaskAssignmentSelectorRegistry

Commit:
LPS-171146 Simplify logic by using method rather than osgi property for NotificationMessageGenerator

Commit:
LPS-171146 Convert method reference to ServiceTrackerMap

Commit:
LPS-171146 SF, inline

Commit:
LPS-171146 Simplify logic by using method rather than osgi property for NotificationSender

Commit:
LPS-171146 Convert method reference to ServiceTrackerMap

Commit:
LPS-171146 SF, inline

Commit:
LPS-171146 SF, rename

Commit:
LPS-171146 Semantic versioning

alessiorendina committed:
COMMERCE-10481 Added grouped product schema and endpoints

alessiorendina committed:
COMMERCE-10481 Regen REST module

alessiorendina committed:
COMMERCE-10481 Regen services

alessiorendina committed:
COMMERCE-10481 Semantic versioning (gw baseline)

alessiorendina committed:
COMMERCE-10481 Refactor commerce product definition local and remote service implementation

alessiorendina committed:
COMMERCE-10481 Added required dependency

alessiorendina committed:
COMMERCE-10481 Fixed grouped commerce product type test util

alessiorendina committed:
COMMERCE-10481 Added grouped product dto converter

alessiorendina committed:
COMMERCE-10481 Added grouped product resource implementation

tinatian committed:
LPS-171358 SF, simplify

tinatian committed:
LPS-171358 Clean up logic as _selectedCategoryIds is always empty

alessiorendina committed:
COMMERCE-10481 Added grouped product exception mappers

alessiorendina committed:
COMMERCE-10481 Added new exception

ccorreagg committed:
LPS-171617 Move logic to a common base class and validate the existence of previous Configuration over both configurations to delete them if it existed previously

ccorreagg committed:
LPS-171617 Add VulcanConfiguration scoped by company

ccorreagg committed:
LPS-171617 Replicate same validations for the Vulcan Configuration scoped by company

ccorreagg committed:
LPS-171617 Register VulcanCompanyConfiguration instead and register for each object independently of the RESTContextPath

ccorreagg committed:
LPS-171617 Use both Vulcan Configurations to get the parameters, scoped and non-scoped

shuyangzhou committed:
LPS-172239 These are clearly wrong.

MarinhoFeliphe committed:
LPS-169623 Use ModelResourcePermissionRegistryUtil

shuyangzhou committed:
LPS-172239 ModelResourcePermissionFactory should not be used in modules.

shuyangzhou committed:
LPS-172239 PortletResourcePermissionFactory should not be used in modules.

MarinhoFeliphe committed:
LPS-172188 User must not be able to create object entries related to account entries that they don't have permission to ADD_OBJECT_ENTRY

MarinhoFeliphe committed:
LPS-172188 Cover by usigin integration tests

shuyangzhou committed:
LPS-172191 Inline

shuyangzhou committed:
LPS-172191 registerContext=true does not do anything when initModuleFramework=false

shuyangzhou committed:
LPS-172295 Initial checkin

shuyangzhou committed:
LPS-172295 Patch to simplify bean property scanning

shuyangzhou committed:
LPS-172295 Apply usage

georgel-pop-lr committed:
LPS-172224 Add portletId to the log

carolmariaabb committed:
LPS-171052 baseline

sirfabios committed:
LPS-171876 Test fix CalendarColor

julschong committed:
LPS-172253 Remove Optional usages from PortalSettingsFormContributor api and implementations

julschong committed:
LPS-172253 Apply to usages.
Since all implementations now all return a simple String, there is no need for null checking

tinatian committed:
LPS-171146 SF, inline

shuyangzhou committed:
LPS-172191 Start UpgradeReportLogAppender as initFramework callback to make sure covering modules startup.

shuyangzhou committed:
LPS-172295 lib versions

julschong committed:
LPS-172253 Auto, gw baseline

gabrielwas committed:
LPS-169697 create hasModelResourcePermission method

gabrielwas committed:
LPS-169697 buildService

gabrielwas committed:
LPS-169697 check if user can see that entry

jeffwu0724 committed:
LPS-172311 remove the usage of optional in SiteNavigationMenuItemTypeContext

jeffwu0724 committed:
LPS-172311 apply usage

jeffwu0724 committed:
LPS-172311 semantic version

shuyangzhou committed:
LPS-172430 Remove unused methods

shuyangzhou committed:
LPS-172430 Inline

shuyangzhou committed:
LPS-172430 Usage shows this is nullable

shuyangzhou committed:
LPS-172430 Rewrite to be ServiceTrackerMap based data structure.

evibudai committed:
LPS-172150 Throw exception when a background task fails

Commit:
LPS-172220 Fix ViewSiteCreationMessage test

Commit:
LPS-170202 Fix name on task

Commit:
LPS-170202 Fix CanRelateOneToManyCustomObjectEntryWithNativeObjectEntries test

tinatian committed:
LPS-171052 SF, rename

tinatian committed:
LPS-171052 SF

Commit:
LPS-172406 Fix tests with Custom Layout

Commit:
LPS-172406 Add skip.portal.instance property to some tests on ObjectCustomLayouts

tinatian committed:
LPS-172224 SF

Commit:
LPS-169697 Make possible the check if user can see that entry during its onAfterCreate phase

shuyangzhou committed:
LPS-172430 Move to static block to avoid unit tests impact.

Commit:
LPS-169697 Remove Feature flag

Commit:
LPS-169697 AFCB

liulukiwi committed:
LPS-169806 Update macro and script for change in LPS-171399

brianchandotcom committed:
LPS-172188 Use StringBundler from petra

brianchandotcom committed:
Prepare for U59

brianchandotcom committed:
LPS-169623 Do not modify TransformUtil unless we really need to

brianchandotcom committed:
LPS-169697 Fix compile errors?

brianchandotcom committed:
LPS-171617 As used

brianchandotcom committed:
LPS-171617 Buy space

brianchandotcom committed:
COMMERCE-10481 Like this

brianchandotcom committed:
COMMERCE-10481 Is this more accurate?

brianchandotcom committed:
Regen portal-impl/src/portal-osgi-configuration.properties

brianchandotcom committed:
LPS-172224 use :

brianchandotcom committed:
LPS-172191 Rename

brianchandotcom committed:
LPS-172295 Rename

brianchandotcom committed:
LPS-172295 prep next

liferay-translation committed:
LPS-77699 Update Translations

Commit:
COMMERCE-9744 - Time values in SubscriptionTypeImpl now also suits plural form

Commit:
COMMERCE-9744 - Added new language for Week(s) and modified another time language keys to adjust their uppercase

Commit:
COMMERCE-9744 - BuildLang

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

brianikim commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 36 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ffaf63c35bea589c743101c834914c4829f43ef5">ffaf63c35bea589c743101c834914c4829f43ef5</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/elocamp/liferay-portal/tree/COMMERCE-9744">COMMERCE-9744</a><br/>Branch GIT ID: <a href="https://github.com/elocamp/liferay-portal/commit/a26ecedc2484782be1f114eea57b975caae0b88f">a26ecedc2484782be1f114eea57b975caae0b88f</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-12.liferay.com/job/test-portal-source-format/4634/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-12.liferay.com/userContent/jobs/test-portal-source-format/builds/4634/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-12.liferay.com/job/test-portal-source-format/4634/">test-portal-source-format#4634</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-12.liferay.com/userContent/jobs/test-portal-source-format/builds/4634/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/36">brianikim#36</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2155005781">[master] ci:test:sf - elocamp &gt; brianikim - PR#36 - 2023-01-10[12:10:12]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-12.liferay.com/job/publish-testray-report/3026/">publish-testray-report#3026</a><br/></div></div>
